From ae61b61fb93efc8b6e86244a7e0ee1fb886b4ede Mon Sep 17 00:00:00 2001
From: Yumi Yukimura <me.cafebabe@gmail.com>
Date: Tue, 2 Jun 2026 16:41:07 +0800
Subject: [PATCH] aidl: fingerprint: Android.bp: Let `compile_multilib` default
 to nothing

When it defaults to "64", while compiling for 32-bit only device
(even if it is NOT using this repository at all), such error appears:

error: hardware/xiaomi/hidl/biometrics/fingerprint/Android.bp:8:1: dependency "libudfpshandlerfactory" of "android.hardware.biometrics.fingerprint@2.3-service.xiaomi" missing variant:
  os:android,image:vendor,arch:arm_armv8-a,link:static
available variants:
  os:android,image:vendor,link:static

Change-Id: I4c731f08a5f65bf0d2b7f2ac30ad68f9576b5699
---
 aidl/fingerprint/Android.bp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/aidl/fingerprint/Android.bp b/aidl/fingerprint/Android.bp
index 3d785ff..b69a9cf 100644
--- a/aidl/fingerprint/Android.bp
+++ b/aidl/fingerprint/Android.bp
@@ -27,7 +27,7 @@ xiaomi_hardware_biometrics_config_default {
     soong_config_variables: {
         RUN_32BIT: {
             conditions_default: {
-                compile_multilib: "64",
+                compile_multilib: "",
             },
             compile_multilib: "prefer32",
         },
-- 
2.47.3

